### ISSUE TITLE
Previous jars are not accessible

### ISSUE THREAD

**Issue by apflieger** [classgraph/classgraph#219]
I was trying to update to 3.1.14, I use SBT so I depend on 
"io.github.lukehutch" % "fast-classpath-scanner" % "3.1.14"
I get an error:

```
sbt.librarymanagement.ResolveException: unresolved dependency: io.github.classgraph#classgraph;4.0.0: not found
```

That's the new redirection I guess.

I finally use the last beta, which works

```
"io.github.classgraph" % "classgraph" % "4.0.0-beta-11"
```

**Comment by lukehutch** [comment]
@apflieger My apologies -- I jumped the gun in redirecting the old Maven artifact to 4.0.0 final. This should be fixed in the next couple of days when I push out 4.0.0 final. Thanks for the report.

**Comment by lukehutch** [comment]
@apflieger I "fixed this" by pushing 3.1.15 that redirects to a beta release that actually exists! Sorry about that. It will still be a surprise to anyone trying to update, when they see that their FastClasspathScanner code won't compile (and that the class doesn't even exist anymore), that the `fast-classpath-scanner` dependency they were expecting to be pulled in isn't there, and that there's a new dependency `classgraph-4.0.0-beta-12` that came out of nowhere...
